src/redux/books/books.js

```diff
@@ -0,0 +1,15 @@
+export default function booksReducer(state = {}, action) {
```

Comment:
- [ ] It would be better if we set the initial state to an empty array `[ ]` instead of the empty object `{}`
![Screenshot from 2023-01-25 21-58-49](https://user-images.githubusercontent.com/46388113/214699769-3f0a5b4f-391f-4f70-9d62-8e26209b411d.png)
Let us remember to set the empty array for our categories as well